ludwig/encoders/sequence_encoders.py

```diff
@@ -92,16 +93,28 @@ def forward(self, input_sequence, mask=None):
         :type mask: Tensor
         """
         input_sequence = input_sequence.type(torch.float32)
+        batch_size = input_sequence.shape[0]
         while len(input_sequence.shape) < 3:
             input_sequence = input_sequence.unsqueeze(-1)
         hidden = self.reduce_sequence(input_sequence)
 
+        # Output may have [batch_size, s, 1] shape, so ensure it is [batch_size, s]
+        hidden = hidden.reshape((batch_size, -1))
```

Comment:
not sure what we decided for sequence and text (probalby to remove the passthrough encoder) but this may not work in some scnarios (happy to elaborate)

Reply:
Hmm, seems so from the tests. Our docs are misleading, then. What do you think is a good workaround in this case?